code/game/gamemodes/shadowling/special_shadowling_abilities.dm

```diff
@@ -155,6 +155,36 @@ var/list/possibleShadowlingNames = list("U'ruan", "Y`shej", "Nex", "Hel-uae", "N
 					M << "<span class='userdanger'>An immense pressure slams you onto the ground!</span>"
 				world << "<font size=5><span class='shadowling'><b>\"VYSHA NERADA YEKHEZET U'RUU!!\"</font></span>"
 				world << 'sound/hallucinations/veryfar_noise.ogg'
+				if(!ticker.mode.wizards)
+					message_admins("A space wizard has spawned to try and combat the Ascendant.")
+					priority_announce("ERROR \[0x0441c\]: MEMORY HIJACK EXECUTED. UNAUTHORIZED ANNOUNCEMENT DETECTED.", null, 'sound/misc/wizard_announce.ogg')
+
+					var/list/mob/dead/observer/candidates = list()
+					var/mob/dead/observer/theghost = null
+
+
+					pollCandidates("Do you wish to be considered for the special position of the Space Wizard researcher?", "wizard", null, FALSE, 100)
+
+					sleep(100)
```

Comment:
DEEP LORE reasoning: the lights out is caused by the anti-interaction of the abstraction-containment super soulshard knife vibrating against the lambda wave produced by the ascendant's astral powers, resulting in a localized EMP.